python/dgl/convert.py

```diff
@@ -1688,36 +1793,54 @@ def to_networkx(g, node_attrs=None, edge_attrs=None):
     >>> nx_g.edges(data=True)
     OutMultiEdgeDataView([(1, 1, {'id': 0, 'h1': tensor([1.]), 'h2': tensor([0., 0.])}),
                           (2, 3, {'id': 1, 'h1': tensor([1.]), 'h2': tensor([0., 0.])})])
+
+    With a heterogeneous graph:
+
+    >>> g = dgl.heterograph({
+        ...     ('user', 'follows', 'user'): (torch.tensor([0, 1]), torch.tensor([1, 2])),
```

Comment:
"..." needs to be aligned with ">>>". Same for all the usage below.

Reply:
You're right, I'm not sure why that happened. Fixed.